### PR TITLE
https://issues.redhat.com/browse/ACM-11407 Add URL custom. docs

### PR DESCRIPTION
**Pull request by oafischer**
Train 14, 2.11 only

### CODE REVIEW

**Review by douglascamata — CHANGES_REQUESTED** [review]
@oafischer I would like to add some more details to the these docs:

1. `customObservabilityHubURL` and `customAlertmanagerHubURL` only support `https` URLs and will automatically prepend the `https://` scheme if it's not present. `http` is not supported. 
2. `customObservabilityHubURL` may include the standard path for the Remote Write API (`/api/metrics/v1/default/api/v1/receive`). If it's not included, ACM Observability will automatically add it at runtime.

I'm replicating this information in the ticket.

**Comment by oafischer** [comment]
Updated, thanks for the additional info @douglascamata. Feel free to review when ready.

File: observability/customize_observability.adoc [hunk 1]
```diff
@@ -472,6 +472,25 @@ If your development needs change, you might need to disable the proxy setting fo
...
+spec:
+  advanced:
+    customObservabilityHubURL: <yourURL>
+    customAlertmanagerHubURL: <yourURL>
+----
+
+*Notes*:
+
+* Only HTTPS URLs are supported. If you do not add `https://` to your URL, the scheme is added automatically.
+* You can include the standard path for the Remote Write API `/api/metrics/v1/default/api/v1/receive` in the `customObservabilityHubURL` spec. If you do not include the path, Observability automatically adds the path at runtime.
```
Comment (dockerymick) [inline observability/customize_observability.adoc]:
```suggestion
* You can include the standard path for the Remote Write API, `/api/metrics/v1/default/api/v1/receive` in the `customObservabilityHubURL` `spec`. If you do not include the path, the Observability service automatically adds the path at runtime.
```

**Review by dockerymick — APPROVED** [review]
Small suggestions. Otherwise, this looks great to me!

**Review by douglascamata — APPROVED** [review]
/lgtm

**Pull request merged**